### ISSUE TITLE
Editorial: [Keep/Drop, Simplify, Expand, Re-write, Re-organize, Move] 

### ISSUE THREAD

**Issue by PsySc0rpi0n** [lnbook/lnbook#994]
**Which part needs editing?**
Which part of the book needs editing? Filename, line?
Filename:
 [04_node_client.asciidoc](https://github.com/lnbook/lnbook/blob/develop/04_node_client.asciidoc)
Section:
https://github.com/lnbook/lnbook/blob/develop/04_node_client.asciidoc#the-lnd-docker-container

**What editing does this need?**
Some options: Keep/Drop, Simplify, Expand, Re-write, Re-organize, Move
Update of some of the commands used

**How does it currently read?**
Copy the text that needs editing, if it is small enough
`$ docker pull lnbook/lnd`
However, this command rerurns this error:
```
$ docker pull lndbook/lnd
Using default tag: latest
Error response from daemon: pull access denied for lndbook/lnd, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

**How should it read?**
Write a recommended solution here, if you can!
From dockerhub I found this command:
`docker pull lncm/lnd`
However it also errors out, not sure due to missing dependencies or what.
I'll post part of the error. It is huge output:
```
 ---> Running in 7dec28af7b43
github.com/lightningnetwork/lnd (download)
github.com/btcsuite/btcd (download)
github.com/decred/dcrd (download)
get "golang.org/x/crypto/sha3": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at //golang.org/x/crypto/sha3?go-get=1
get "golang.org/x/crypto/sha3": verifying non-authoritative meta tag
golang.org/x/crypto (download)
...
...
...
github.com/tv42/zbase32 (download)
get "golang.org/x/crypto/acme/autocert": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at //golang.org/x/crypto/acme/autocert?go-get=1
get "golang.org/x/crypto/acme/autocert": verifying non-authoritative meta tag
The command '/bin/sh -c mkdir -p ${GOPATH}/src && 	cd ${GOPATH}/src && 	go get -v -d github.com/lightningnetwork/lnd && 	cd ${GOPATH}/src/github.com/lightningnetwork/lnd && 	git checkout tags/${LND_VER} && 	make clean && make && make install' returned a non-zero code: 1
```

**Comment by aantonop** [comment]
I see "lnbook/lnd" in the text but "lndbook/lnd" in the command you tried.
Check that?

**Comment by PsySc0rpi0n** [comment]
> I see "lnbook/lnd" in the text but "lndbook/lnd" in the command you tried. Check that?

Indeed. I'm sorry for my distraction.

However, now, when I try to build lnd inside docker I get errors. Will it be because I tried the other command (docker pull lncm/lnd) ???
```
poRoot:"https://go.googlesource.com/crypto"} at //golang.org/x/crypto/acme/autocert?go-get=1
get "golang.org/x/crypto/acme/autocert": verifying non-authoritative meta tag
The command '/bin/sh -c mkdir -p ${GOPATH}/src && 	cd ${GOPATH}/src && 	go get -v -d github.com/lightningnetwork/lnd && 	cd ${GOPATH}/src/github.com/lightningnetwork/lnd && 	git checkout tags/${LND_VER} && 	make clean && make && make install' returned a non-zero code: 1
```
Thanks
Psy

**Comment by PsySc0rpi0n** [comment]
Issue closed with #995
